### PR TITLE
feat: 平台信息支持所属BG #8259

### PR DESCRIPTION
**Pull request by Mr-SakuraManJi**
feat: 平台信息支持所属BG #8259

### CODE REVIEW

File: support-files/sql/2008_ci_store-update_v1.9_mysql.sql [hunk 1]
```diff
@@ -108,14 +108,6 @@ BEGIN
...
     END IF;
 
-    IF NOT EXISTS(SELECT 1
-            FROM information_schema.COLUMNS
-            WHERE TABLE_SCHEMA = db
-                AND TABLE_NAME = 'T_STORE_DOCKING_PLATFORM'
-                AND COLUMN_NAME = 'OWNERS') THEN
-        ALTER TABLE T_STORE_DOCKING_PLATFORM ADD COLUMN `OWNERS` varchar(1024) NOT NULL COMMENT '运营负责人';
-    END IF;
-
```
Comment (carlyin0801) [inline support-files/sql/2008_ci_store-update_v1.9_mysql.sql:118]: 改错了，应该加上如果存在这个字段就要删掉的sql

**Comment by tencentblueking-adm** [comment]
[CLA assistant check](https://bk-cla.bktencent.com:443/TencentBlueKing/bk-ci?pullRequest=8337) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://bk-cla.bktencent.com:443/TencentBlueKing/bk-ci?pullRequest=8337) before we can accept your contribution.<br/>**0** out of **2** committers have signed the CLA.<br/><br/>:x: Mr-SakuraManJi<br/>:x: yjieliang<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://bk-cla.bktencent.com:443/check/TencentBlueKing/bk-ci?pullRequest=8337) it.</sub>

**Review by carlyin0801 — APPROVED** [review]
ok

**Review by irwinsun — APPROVED** [review]
内部考虑不影响，上线后不会有操作新增（由运营控制），否则OWNER NOT NULL 会报错

**Pull request merged**